docs/coupled/index.md

```diff
@@ -0,0 +1,64 @@
+[E3SM step-by-step guide](https://docs.e3sm.org/running-e3sm-guide/).
+
+## Scientifically supported coupled compsets and grids
+
+### Compsets
+
+Coupled compsets consist of interactive prognostic Earth system components under specific forcing conditions.
+Coupled compsets in E3SM are developed  for three science-driven simulation campaigns:
+
+1) water cycle change and impacts, 2) human-earth system feedbacks, and 3) polar processes, sea-level rise and coastal impacts.
+
+The standard coupled configurations -- which consist of active atmosphere, land, ocean and sea-ice components -- form the base physical
+coupled system and are mainly designed for water cycle change and impacts simulation campaign.
+The compsets for such configurations are listed below.
+
+|Compset alias | Description |
+|:-----------  |:----------- |
+|`WCYCL1850` | Standard configuration with pre-industrial climatological forcings |
+|`WCYCL1850-4xCO2` | Same as `WCYCL1850` except with abrupt (then persistent) 4xCO2 forcing. |
+|`WCYCL1850-1pctCO2` | Same as `WCYCL1850` except with 1 percent per year increase of CO2 concentration |
+|`WCYCL1950` | Standard configuration with perpetual 1950 forcings |
+|`WCYCL20TR` | Standard configuration with prescribed transient forcings over the historical period (1850-2014) |
+|`WCYCLSSP245` | Standard configuration with prescribed SSP-245 forcings |
+|`WCYCLSSP370` | Standard configuration with prescribed SSP-370 forcings |
+|`WCYCLSSP585` | Standard configuration with prescribed SSP-585 forcings |
+
+The E3SMv3 compsets for the other two science simulation campaigns are being finalized,
+with additional components and/or features. The compset naming follows the same convention, e.g., `CRYO1850` and `CRYO1850-4xCO2` are with prognostic ice-shelf melt fluxes for polar processes simulation campaign.
+
+### Grids
+
+|Compset alias | Description |
+|:-----------  |:----------- |
+|`ne30pg2_r05_IcoswISC30E3r5` | E3SMv3 standard low-resolution tri-grid, with ne30pg2 (110 km) atmosphere, 0.5deg x 0.5deg land and river, and Icosahedral 30 km ocean/seaice mesh with ice shelves cavities (wISC), E3SMv3 (E3) revision r5 |
+|`northamericax4v1pg2_r025_IcoswISC30E3r5` |  North America regionally refined (110 to 25 km) atmosphere, 0.25deg x 0.25deg land and river, and the same ocean/seaice mesh for the low-resolution tri-grid. |
+|`ne120pg2_r025_IcoswISC30E3r5` |  High-resolution tri-grid, with ne120pg2 (25 km) atmosphere, 0.25deg x 0.25deg land and river. The ocean and seaice mesh to be updated with a variable 18 to 6 km mesh. |
+
+## Input data
+
+Inputdata for coupled compsets at component model levels are the same as for the standalone compnent configurations
+for a given forcing scenario (e.g., `1850` for the pre-industrial period,  `20TR` for the historical period, `2010`
+for present-day condition, and `SSPs` for Shared Socioeconomiuc Pathways of climate change scenarios).
```

Comment:
typo: 'Socioeconomiuc' should be 'Socioeconomic'

Reply:
Thanks, Chris. This will be fixed in a new PR.